tests/external/test_gcs_crashstorage_with_emulator.py

```diff
@@ -0,0 +1,119 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Why put this test file in `tests/external/` instead of with the other tests in `tests/unittest/`?